libs/community/langchain_community/llms/bedrock.py

```diff
@@ -335,11 +335,13 @@ def validate_environment(cls, values: Dict) -> Dict:
                 "Could not import boto3 python package. "
                 "Please install it with `pip install boto3`."
             )
+        except ValueError as e:
+            raise ValueError(f"Error raised by bedrock service: {e}")
```

Comment:
should we just reraise?
```
except ValueError as e:
    raise e
```

Reply:
Wouldn't it be more transparent, that the error is not Langchain native, but side-service originated?

The exact error wording is "Did not find region_name, please add an environment variable `AWS_DEFAULT_REGION` which contains it, or pass `region_name` as a named parameter"